lib/miq_expression/tag.rb

```diff
@@ -58,6 +51,10 @@ def arel_attribute
     target && target.arel_table[:id, arel_table]
   end
 
+  def valid?
+    !!target
+  end
```

Comment:
What is the difference between valid? returning false and an InvalidTarget?

Reply:
the syntax for a tag is valid. But the reference in value is invalid.
(syntax ok, semantics bad)